data/command.go

```diff
@@ -40,15 +40,36 @@ func (c CommandParameters) String() string {
 // a chat provider.
 type CommandRequest struct {
 	CommandEntry
-	Adapter    string            // The name of the adapter this request originated from
-	ChannelID  string            // The provider ID of the channel that the request originated in
-	Context    context.Context   // The request context
-	Parameters CommandParameters // Tokenized command parameters
-	RequestID  int64             // A unique requestID
-	Timestamp  time.Time         // The time this request was triggered
-	UserID     string            // The provider ID of user making this request
-	UserEmail  string            // The email address associated with the user making the request
-	UserName   string            // The gort username of the user making the request
+
+	// Adapter is the name of the adapter this request originated from
+	Adapter string
+
+	// ChannelID is the (provider-specific) ID of the channel that the
+	// request originated in
+	ChannelID string
+
+	// Context is the original request context
+	// TODO How can we refactor this out?
```

Comment:
Do we have an issue for this?

Reply:
No. I wasn't sure we really needed one, but I'd be happy to create one if you think it's a good idea!